client.cpp

```diff
@@ -490,3 +491,17 @@ PHP_METHOD(VoltClient, drain)
         RETURN_FALSE;
     }
 }
+
+PHP_METHOD(VoltClient, close)
+{
+    zval *zobj = getThis();
+    voltclient_object *obj = (voltclient_object *)zend_object_store_get_object(zobj TSRMLS_CC);
+
+    try {
+        voltdb::ConnectionPool::pool()->closeClientConnection(*(obj->client));
+    } catch (voltdb::MisplacedClientException) {
+        zend_throw_exception(zend_exception_get_default(TSRMLS_C), NULL, errMisplacedClientException TSRMLS_CC);		
+        RETURN_FALSE;
+    }
+    RETURN_NULL();
```

Comment:
The return type is mixed up. Either this method returns a boolean, or it returns some object or none at all. You return `false` in the exception handler, but returning NULL here.